react/src/views/Box/BoxView.tsx

```diff
@@ -333,7 +362,21 @@ function BTBox() {
         newLocationId: parseInt(locationId, 10),
       },
       refetchQueries: [refetchBoxByLabelIdentifierQueryConfig(labelIdentifier)],
```

Comment:
It's not related to the direct changes of this PR but I was wondering why we refetch here. It's additional overhead whereas the new location could already be returned as response from the `updateBoxLocation` in the first place.

Reply:
@pylipp You are completely right, however, trying to remove the refetch did not update the box state anymore, so I need to investigate further why it was originally added.

Reply:
Thanks! I don't think it's super pressing but probably not an ideal solution either.